components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/AbstractLocalApplicationAuthenticator.java

```diff
@@ -85,8 +86,9 @@ public AuthenticatorFlowStatus process(HttpServletRequest request, HttpServletRe
                 } catch (AuthenticationFailedException e) {
                     if (isAccountLocked(context)) {
                         try {
-                            String redirectUrl = getRedirectUrlOnAccountLock(context, response);
-                            response.sendRedirect(redirectUrl);
+                            String status = FrameworkConstants.ACCOUNT_LOCKED_MSG;
```

Comment:
If getRedirectUrlOnAccountLock method is not used anywhere else, let's remove it